### PR TITLE
Package dependencies

### PR DESCRIPTION
**Pull request by huntermontell**

### CODE REVIEW

File: tests/Feature/Nova/ProductModelTest.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    use DatabaseTransactions;
+    use WithFaker;
+
+    /** @test */
+    public function create()
+    {
+        $model = Product::factory()->create();
+        $this->assertNotNull($model);
+    }
+}
```
Comment (drewroberts) [inline tests/Feature/Nova/ProductModelTest.php]: Thanks for creating this test! The only change is to put it in a different folder.

Comment (drewroberts) [reply tests/Feature/Nova/ProductModelTest.php]: It already exists in the correct folder:

https://github.com/tipoff/products/blob/main/tests/Unit/Models/ProductModelTest.php

Comment (huntermontell) [reply tests/Feature/Nova/ProductModelTest.php]: Ahh of course

**Review by drewroberts — APPROVED** [review]
Thank you! 🎻

**Pull request merged**